src/components/Block/Block.js

```diff
@@ -25,7 +25,11 @@ const Block = ({
 };
 
 Block.propTypes = {
-  is: PropTypes.string
+  is: PropTypes.string,
+  mt: PropTypes.string,
+  mb: PropTypes.string,
+  ml: PropTypes.string,
+  mr: PropTypes.string
```

Comment:
@Cooperbuilt Do we want to enforce `defaultProp` declarations if a prop is optional?